cost-analyzer/templates/_helpers.tpl

```diff
@@ -1002,6 +1008,12 @@ Begin Kubecost 2.0 templates
     {{- toYaml .Values.kubecostAggregator.extraVolumeMounts | nindent 4 }}
     {{- end }}
   env:
+    {{- if .Values.kubecostAggregator.useInMemoryIPC }}
+    - name: IPC_DIR
+      value: /ipc1
+    - name: PARQUET_DIR
```

Comment:
jesse, I'd suggest that these go under separate flags, and while IPC_DIR is on by default, parquet is off. We have an idea of the bound of the IPC_CIR, but parquet dir could be unbounded, more detail here: https://kubecost.atlassian.net/browse/ENG-1232?focusedCommentId=17994